experimental/LieAlgebras/src/RootSystem.jl

```diff
@@ -274,7 +288,7 @@ This is a more efficient version for `positive_coroots(R)[i]`.
 Also see: `positive_coroots`.
 """
 function positive_coroot(R::RootSystem, i::Int)
-  return R.positive_coroots[i]::DualRootSpaceElem
+  return (R.positive_coroots::Vector{DualRootSpaceElem})[i]
```

Comment:
How about this to avoid the repeated type annotations? (But I am fine with this either way)
```suggestion
  return positive_coroots(R)[i]
```

Reply:
for the positive (co)roots this indeed works. For the negative ones, however, this is not optimal as it would first negate all positive roots to then only take one of them. And I rather have positive and negative consistent here.